scripts/kconfig/kconfigfunctions.py

```diff
@@ -574,6 +591,7 @@ def shields_list_contains(kconf, _, shield):
         "dt_chosen_label": (dt_chosen_label, 1, 1),
         "dt_chosen_enabled": (dt_chosen_enabled, 1, 1),
         "dt_chosen_path": (dt_chosen_path, 1, 1),
+        "dt_chosen_has_compat": (dt_chosen_has_compat, 2, 2),
```

Comment:
Please document this in `doc/build/kconfig/preprocessor-functions.rst`

Reply:
Added to the documentation

Reply:
> Please document this in doc/build/kconfig/preprocessor-functions.rst

I was unaware of this file. Thanks for catching that, and I will make a note to request the same change in the future